controllers/articles-controller.js

```diff
@@ -22,3 +24,32 @@ exports.patchArticleVotesById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getArticles = (req, res, next) => {
+  const { topic } = req.query;
+  selectArticles(topic)
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch(next);
+};
+
+exports.getArticleComments = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticleComments(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
+    })
+    .catch(next);
+};
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  console.log(req.body);
+  addComment(article_id, username, body)
+    .then((addedComment) => {
+      res.status(201).send({ addedComment });
```

Comment:
We want our new comment to have a key of "comment", as its best practices to stay consistent - makes life a lot easier on the Front-End side of things!